test/beast/http/vector_body.cpp

```diff
@@ -18,6 +19,12 @@ BOOST_STATIC_ASSERT(is_body<vector_body<char>>::value);
 BOOST_STATIC_ASSERT(is_body_writer<vector_body<char>>::value);
 BOOST_STATIC_ASSERT(is_body_reader<vector_body<char>>::value);
 
+#if __cpp_lib_byte >= 201603
```

Comment:
is there a boost macro for this? and if so, should we use it instead? ping @glenfe

Reply:
I'm hearing that it is okay, this will go into v253.